www/benj12/css/styles.css

```diff
@@ -0,0 +1,104 @@
+body {
+	font-family: Arial, Helvetica, sans-serif;
+}
+.nametext{
+	color: red;
+}
+.rgbtext{
+	color:rgb(0,100,0);
+}
+.hexboldtext{
+	color:#00BFFF;
+	font-weight: bold;
+}
+.text18{
+	font-size: 18px;
+}
+.text24{
+	font-size: 24px;
+	font-style: italic;
+}
+.whitebrowntext{
+	color:white;
+	background-color: brown;
+}
+.centerline{
+	text-align: center;
+	line-height: 32px;
+}
+.justified{
+	text-align: justify;
+	text-transform: capitalize;
+}
+.box1{
+	width: 400px;
+	height: 300px;
+	border-style: solid;
+	border-width: 1px;
+	border-color: purple;
+	padding: 30px;
+}
+.box2{
+	width: 500px;
+	height: 200px;
+	background-color: red;
+	margin: 100px;
+}
+.hidden{
+	display: none;
+}
+.box3{
+	width: fit-content;
+	height: 150px;
+	background-color: #00BFFF;
+	padding: 50px;
+	border-style: solid;
+	border-width: 10px;
+	border-color: yellow;
+}
+.hiddengap{
+	visibility: hidden;
+}
+.box4{
+	width: 200px;
+	height: 500px;
+	background-color: green;
+	opacity: 0.5;
+}
+.box5{
+	width: 400px;
+	height: 300px;
+	position: relative;
+	background-color: lightblue;
+}
+.box6{
+	width: 50px;
+	height: 50px;
+	position: absolute;
+	background-color: pink;
+	top: 50px;
+	right: 10px;
+}
+.button1{
+	width: 300px;
+	height: 100px;
+}
+.button1:active{
+	color: blue;
+}
+.button2{
+	width: 200px;
+	height: 50px;
+}
+.button2:hover{
+	background-color: brown;
+}
+.box7{
+	margin-top: 200px;
+	width: 200px;
+	height: 500px;
+	background-color: blue
+}
+.box7:hover{
+	cursor: pointer;
```

Comment:
vlastnost cursor nema byt v :hover protoze sama jiz implikuje ze element musi byt hovered aby to melo efekt